build_tools/cmake/build_docs.sh

```diff
@@ -58,3 +59,61 @@ scripts/update_op_coverage.py ${BUILD_DIR}
 # Update e2e_coverage.md
 PYTHON_BIN=`which python3` scripts/update_e2e_coverage.py ${BUILD_DIR}
 
+# Generate IR conversion flow examples
+
+# The list of input IR examples
+declare -a ir_examples=(
+  iree/samples/ops/mhlo-dot.mlir
+  iree/samples/ops/mhlo-conv.mlir
+)
+# The list of compiler targets
+declare -a compiler_targets=(
+  llvm-ir
+  vmla
+  vulkan-spirv
+)
+
+mkdir -p ${BUILD_DIR}/doc/ir_examples
+for example in "${ir_examples[@]}"; do
+  for target in "${compiler_targets[@]}"; do
+    example_name=$(basename ${example%.mlir})
+    filepath=${BUILD_DIR}/doc/ir_examples/${example_name}-${target}.md
+    title="${example_name} IR conversion example for ${target} backend"
```

Comment:
These titles are pretty long. Since they're already nested on "IR Conversion Examples", they could be abbreviated. How about a format like "mhlo-conv on vulkan-spirv"?

Reply:
Sure.